core/scripts/functions/src/generate_ocr2_config_cmd.go

```diff
@@ -19,11 +20,36 @@ import (
 	"github.com/smartcontractkit/chainlink/core/services/ocr2/plugins/directrequestocr/config"
 )
 
+type InputConfig struct {
+	MaxQueryLengthBytes       uint32
```

Comment:
Let's either add a comment that this is a json thing and fields must not be renamed, or use `json:""` attributes.